templates/zerver/integrations.html

```diff
@@ -2062,6 +2070,85 @@
 
     <div id="trello" class="integration-instructions">
 
+      <p>
+        First, create the stream you'd like to use for Trello notifications,
+        and subscribe all interested parties to this stream. We recommend the
+        name <code>trello</code>.
+      </p>
+
+      <p>
+        Your webhook URL is:<br/>
+        <code>https://api.zulip.com/v1/external/trello?api_key=abcdefgh&stream=trello</code><br/>
+        where <code>api_key</code> is the API key of your Zulip bot,
+          and <code>stream</code> is the stream name you want the notifications sent to.
+      </p>
+
+      <p>
+        Before you'll be able to create a Webhook,
+        you need to follow steps below to get an <b>APPLICATION_KEY</b> and a <b>UserToken</b>.
+        Also we need to fetch the <b>idModel</b> of the board.
+      </p>
+
+      <p>
+        To generate the <b>APPLICATION_KEY</b> open this URL in your web browser:<br/>
+        <code>https://trello.com/1/appkey/generate</code>.
+      </p>
+
+      <p>
+        To generate read access token, fill in and open this URL in the browser while being logged into your Trello account:
+        <code>https://trello.com/1/authorize?key=&ltAPPLICATION_KEY&gt&name=Issue+Manager&expiration=never&response_type=token&scope=read</code><br/>
+        You will receive the <b>UserToken</b>, save it.
+      </p>
+
+      <p>
+        From board URL get <b>TRELLO_BOARD_SHORT_ID</b>.
+        URL should look like:<br/>
+        <code>https://trello.com/b/TRELLO_BOARD_SHORT_ID/boardName</code>.
+      </p>
+
+      <p>
+        Now you have: <b>APPLICATION_KEY</b>, <b>UserToken</b>, <b>TRELLO_BOARD_SHORT_ID</b>.<br/>
+        Go to your web browser and open filled URL:<br/>
+        <code>https://api.trello.com/1/board/&ltTRELLO_BOARD_SHORT_ID&gt?key=&ltAPPLICATION_KEY&gt&token=&ltUserToken&gt</code><br/>
+        From returned JSON get <b>id</b> value. It's your <b>idModel</b>, save it.
```

Comment:
I suggest:
`You'll receive some JSON. Within that, find the <b>id</b> value. That's your <b>idModel</b>; note it.`